Unplottable Attributes

apaat opened the pull request:
-Removed multiple attributes that were unplottable. I had removed many of these a while ago but they somehow got readded. The attributes were: HEX_STATUS, HEX_TWEAK, ROT_ENABLED, ADC_HOME1, ADC_HOME2, DATE_OBS, S2n, TRANSPAR, SKYLEVEL, TOWER_TIMESTAMP, SHUTTER_LOWER and SHUTTER_UPPER
-Split the FOCUS attributes into its six contitutent parts. These are currently named FOCUS0, FOCUS1, FOCUS2, FOCUS3, FOCUS4, and FOCUS5

apaat committed:
Unplottable Attributes

-Removed multiple attributes that were unplottable. I had removed many of these a while ago but they somehow got readded. The attributes were: HEX_STATUS, HEX_TWEAK, ROT_ENABLED, ADC_HOME1, ADC_HOME2, DATE_OBS, S2n, TRANSPAR, SKYLEVEL, TOWER_TIMESTAMP, SHUTTER_LOWER and SHUTTER_UPPER
-Split the FOCUS attributes into its six contitutent parts. These are currently named FOCUS0, FOCUS1, FOCUS2, FOCUS3, FOCUS4, and FOCUS5

parfa30 commented:
Looks good. I found the new names, which should be useful. Make this quick change and I'll merge.
FOCUS0 --> FOCUS_X
FOCUS1 --> FOCUS_Y
FOCUS2 --> FOCUS_Z
FOCUS3 --> FOCUS_TIP
FOCUS4 --> FOCUS_TILT
FOCUS5 --> FOCUS_ROT

apaat committed:
Fixed Focus Attribute Names

FOCUS0 -> FOCUS_X
FOCUS1 -> FOCUS_Y
FOCUS2 -> FOCUS_Z
FOCUS3 -> FOCUS_TIP
FOCUS4 -> FOCUS_TILT
FOCUS5 -> FOCUS_ROT

apaat commented:
Alright that is done

On Fri, 06 Nov 2020 10:43:05 -0800
  Parker Fagrelius <notifications@github.com> wrote:
> Looks good. I found the new names, which should be 
>useful. Make this quick change and I'll merge.
>FOCUS0 --> FOCUS_X
>FOCUS1 --> FOCUS_Y
>FOCUS2 --> FOCUS_Z
>FOCUS3 --> FOCUS_TIP
>FOCUS4 --> FOCUS_TILT
>FOCUS5 --> FOCUS_ROT
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/desihub/desiperf/pull/70#issuecomment-723238968
